vstruct/primitives.py

```diff
@@ -724,3 +724,27 @@ def __repr__(self):
         base = "{%.8x-%.4x-%.4x-%.2x%.2x-%.2x%.2x%.2x%.2x%.2x%.2x}"
         return base  % self._guid_fields
 
+class enum_mixin(object):
```

Comment:
makes sense. i'll update this patch to change the constructor to `v_number`.

any thoughts on how a user might fetch the enum instance from a number instance? `v_number.vsGetEnum()`?